### PR TITLE
adding support for HA clusters, using HAProxy

### PR DESCRIPTION
**Pull request by dklyle**
Signed-off-by: David Lyle <dklyle0@gmail.com>

### CODE REVIEW

File: clr-k8s-examples/README.md [hunk 1]
```diff
@@ -63,6 +63,28 @@ devicemapper storage. This should not be used for production.
 
 > NOTE: This step is done automatically if using vagrant.
 
+### For HA, setup the load balancer node
+
+Ideally, the load balancer node will be a separate node. However, one of the
+master nodes can also server as the load balancer for the cluster. [HAProxy](https://www.haproxy.org/)
```
Comment (grahamwhaley) [inline clr-k8s-examples/README.md:69]: s/server/serve/

Comment (dklyle) [reply clr-k8s-examples/README.md]: pushed a commit to fix this

File: clr-k8s-examples/create_stack.sh [hunk 2]
```diff
@@ -14,6 +14,8 @@ SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
 : ${MASTER_IP:=}
 : ${CERT_SANS:=}
 HIGH_POD_COUNT=${HIGH_POD_COUNT:-""}
+LOAD_BALANCER_IP=${LOAD_BALANCER_IP:-""}
+LOAD_BALANCER_PORT="${LOAD_BALANCER_PORT:-6443}"
```
Comment (ganeshmaharaj) [inline clr-k8s-examples/create_stack.sh:18]: Should this be `6444` instead of `6443` which is what k8s master listens on?

Comment (dklyle) [reply clr-k8s-examples/create_stack.sh]: yes, added another commit

**Pull request merged**